### DISCUSSION TITLE
[REQUEST] source map

### DISCUSSION THREAD

**Discussion by damartripamungkas** [yao-pkg/pkg#21] [Ideas]
I looked at the executable file and the last line says source map which contains the file path, is it possible to delete this? if so that's very good in my opinion.

**Comment by robertsLando** [comment]
An easy way would be to pre-build you application using esbuild to create a single file bundle, this will also make it much more small in size

**Reply by damartripamungkas** [reply]
> An easy way would be to pre-build you application using esbuild to create a single file bundle, this will also make it much more small in size 

My script uses worker_thread for some cases, it's not easy to handle that if I bundle it

**Reply by robertsLando** [reply]
I'm open to accept PR to make this an option
